anchor/common/bls_lagrange/src/blst.rs

```diff
@@ -36,7 +23,7 @@ impl TryFrom<u64> for KeyId {
         if value != 0 {
             unsafe {
                 let mut id = blst_scalar::default();
-                blst_scalar_from_uint64(&mut id, &value);
+                blst_scalar_from_le_bytes(&mut id, &value.to_le_bytes()[0], 8);
```

Comment:
This works as is, though I'd consider caching the u64 -> bytes and using the length of bytes rather than hard coding. e.g.

```rust
                let value_le_bytes = value.to_le_bytes();
                blst_scalar_from_le_bytes(&mut id, key_ley_bytes.as_ptr(), value_le_bytes.len());
```